samples/custom_dispatch/vulkan/shaders/example_apply.mlir

```diff
@@ -0,0 +1,55 @@
+// RUN: iree-compile %s \
```

Comment:
naming nit: `example_patterns.mlir` + `example_pattern_module.mlir` so it's clearer the two files are related and they are doing pattern stuff (or `example_pdl.mlir` / `example_pdl_module.mlir`, etc)